Getting-Started/linux.md

```diff
@@ -1,81 +1,68 @@
 
-Now that you've got a Linux environment to play around in, it's time to get our hands dirty. Open up your terminal and let's get started.
+Now that you've got a Linux environment to play around in, it's time to get your hands dirty and work with some commands. For most instances, you will only have to know a core group of commands, as well as have the ability to look up unfamiliar ones by either using the terminal, or your preferred web browser.
+
+The web has thousands of Linux guides that can teach you everything you need to know. Right now, let's focus on the basics. Walk through [this command overview](https://www.hostinger.com/tutorials/Linux-commands) or one you find on Google on a local terminal. Again, this is not meant to be complicated. We just want you to get familiar with the _core group_ of commands. Practice a few times and learn to recognize the outputs of each command. It can be a bit daunting if you've never worked with a terminal environment before, but remember: _the man page is your friend_. Once you feel comfortable with the majority of the commands, move on to the next portion of this guide.
 
-#### pwd
+## Further Configuration
 
-This is a great first command. The `pwd` command will tell you where you are in the file system. If you opened up a terminal in your 'home' directory, your output should look something like this: `/home/cpeters`. This is the path to the current working directory, or folder in layman's terms (pwd stands for Present Working Directory). This output is also known as an absolute path, meaning that it starts with the root directory (/) and continues forward to your current position.
+### VSCode
 
-#### cd
+VSCode is awesome. Simply put, it is the best IDE (Integrated Development Environment) around. If you've taken any sort of coding class in the past, you might have messed around with programs like Eclipse or NetBeans. Throw those out the window. Once you go to VSCode, you'll never go back.
 
-As the name suggests, `cd` stands for "Change Directory" (this will be a large topic, encompassing file paths, . and .. ) Try changing directories to your Desktop directory by typing `cd Desktop`. Keep in mind the shell is case sensitive, so be sure to replicate capitalization.
+On your Linux machine, go to the [VSCode website](https://code.visualstudio.com/download) and download the `.deb` file. Change directories to your Downloads (`cd Downloads`) and use the `dpkg` command with the `-i` suffix to install VSCode (`dpkg -i PACKAGE_NAME`). To launch VSCode, simply switch to a directory with a project and type `code .` which will launch VSCode in the current directory.
 
-#### But wait, I don't have to type the entire name?
+_Now that they have it installed, do we want to walk them through extensions?_
 
-No you don't. Type `cd ..` and then `cd Dow[and then press tab]`. This should complete your entry to `cd Downloads/`. Hit enter and you should be in your Downloads folder. Pretty powerful, right?
+### Anaconda
```

Comment:
This shouldn't be here. Python guide covers this

Reply:
Deleted